.github/workflows/cd.yml

```
@@ -0,0 +1,84 @@
+    environment: Deployment
+    runs-on: ubuntu-22.04
+    needs: publish
+    steps:
+      - name: Install dependencies
+        run: sudo apt-get install node-semver -y
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GIT_TOKEN }}
+      - name: Apply version and date in CHANGELOG.md file
+        run: sed -i "s/^\#\# \[Unreleased\] - yyyy-mm-dd$/\#\# [${{needs.check-version.outputs.version}}] - $(date +'%Y-%m-%d')/g" CHANGELOG.md
```

actionlint (1.7.4)

property "check-version" is not defined in object type {publish: {outputs: {}; result: string}}

(expression)